src/m1_facilitator/repositories/Repositories.ts

```diff
@@ -16,12 +16,15 @@
 
 import { InitOptions, Sequelize } from 'sequelize';
 
+import WithdrawIntentRepository from './WithdrawIntentRepository';
```

Comment:
Please sort the imports, 
Similarly, sort the statements in notify() and constructor()

Reply:
Done.